### PR TITLE
Describe the function used to transform listencounts

### PR DESCRIPTION
**Pull request by amCap1712**
Include a description of what confidence function is being used in the html report generated for summarizing a training model run.

### CODE REVIEW

File: listenbrainz_spark/recommendations/recording/tests/test_models.py [hunk 1]
```diff
@@ -22,11 +22,11 @@ def tearDownClass(cls):
         super(TrainModelsTestCase, cls).delete_dir()
 
     def test_parse_dataset(self):
-        row = Row(spark_user_id=1, recording_id=2, count=3)
+        row = Row(spark_user_id=1, recording_id=2, playcount=3, transformed_listencount=5)
         rating_object = train_models.parse_dataset(row)
         self.assertEqual(rating_object.user, 1)
         self.assertEqual(rating_object.product, 2)
-        self.assertEqual(rating_object.rating, 3)
+        self.assertEqual(rating_object.rating, 5)
```
Comment (mayhem) [inline listenbrainz_spark/recommendations/recording/tests/test_models.py:29]: nitpick: add test for transformed_listencount?

File: listenbrainz_spark/recommendations/templates/model.html [hunk 2]
```diff
@@ -21,9 +21,12 @@ <h2>Data preprocessing and model training</h2>
 		<tr>
 			<th>user_id</th>
 			<th>recording_id</th>
-			<th>count</th>
+			<th>playcount</th>
+            <th>transformed_listencount</th>
```
Comment (mayhem) [inline listenbrainz_spark/recommendations/templates/model.html:25]: soooper nitpick: indent.

**Review by mayhem — APPROVED** [review]
Only nitpicks --other wise good!

File: listenbrainz_spark/recommendations/recording/tests/test_models.py (hunk 1 above)
Comment (amCap1712) [reply listenbrainz_spark/recommendations/recording/tests/test_models.py:29]: This is rating object which keeps only one count. So the test here is checking that the count being kept is transformed_listencount and not playcount.

**Pull request merged**